Created icon/chart for new advantage page

nasin opened the pull request:
Hi, this is the static chart/ animated icon for the new Advantage page.

circleci-mu commented:
### File size comparison
No differences found

circleci-mu commented:
## Visual test results - No difference found

---
<details>
<summary>Samples changed</summary><p>

| Change type | Sample |
| --- | --- |
|  Added | samples/highcharts/website/icon-support/demo.js |
|  Added | samples/highcharts/website/static-support/demo.js |

</p>
</details>

oysteinmoseng commented:
Hi @nasin, took a quick a11y look - will these animate in any way, or could they even just be static SVG files? Wondering if we want to enable tooltip at all for these.

nasin commented:
Hi,

For the animated life preserver icon, I removed the tooltips and put logic in the animation to handle reduced motion prefered.

For the static word cloud, I made the tooltips more meaningful instead of showing occurrences, which were mostly 1s. I added the a11y module to chart as well.

I'm not sure what you mean about using static SVG files.

TorsteinHonsi reviewed (approved):
Thanks!
